src/migration/ember-addon/steps/analyze-addon.js

```diff
@@ -24,11 +28,26 @@ function getProjectRootDevDependencies(options) {
 function getPublicEntrypoints(options) {
   const { projectRoot } = options;
 
-  const filePaths = glob.sync('**/*.{js,ts}', {
-    cwd: join(projectRoot, 'addon'),
+  const filePaths = glob.sync('{addon,addon-test-support}/**/*.{js,ts}', {
+    cwd: projectRoot,
   });
 
-  return filePaths.map((filePath) => filePath.replace(/ts$/, 'js'));
+  return filePaths
+    .map((filePath) => {
+      return renameDirectory(filePath, {
+        from: 'addon',
+        to: '',
+      });
+    })
+    .map((filePath) => {
+      return renameDirectory(filePath, {
+        from: 'addon-test-support',
+        to: 'test-support',
+      });
+    })
```

Comment:
TODO: Since file paths come from two different sources, it'd be good to sort the paths.